Makefile

```diff
@@ -219,12 +219,20 @@ android-ports: ##@other Add proxies to Android Device/Simulator
 android-logcat:
 	adb logcat | grep -e StatusModule -e ReactNativeJS -e StatusNativeLogs
 
+_list:
+	@$(MAKE) -pRrq -f $(lastword $(MAKEFILE_LIST)) : 2>/dev/null | awk -v RS= -F: '/^# File/,/^# Finished Make data base/ {if ($$1 !~ "^[#.]") {print $$1}}' | sort | egrep -v -e '^[^[:alnum:]]' -e '^$@$$'
+
+_unknown-startdev-target-%:
+	@ echo "Unknown target device '$*'. Supported targets:"
+	@ make _list | grep "watch-" | sed s/watch-/startdev-/
+	@ exit 1
+
 startdev-%:
```

Comment:
(optional)
what I dislike about this setup right now is that it breaks autocompletion in the shell...
i don't know, maybe just copy-pasting a few job names might lead to a better UX?

Reply:
Just to be clear, autocompletion was broken before this PR, right?

Reply:
yes, and I always hated that fact 😠

Reply:
Fixed.